hera_opm/mf_tools.py

```diff
@@ -1111,20 +1142,34 @@ def build_analysis_makeflow_from_config(
                             print("{0} {1}".format(command, prepped_args), file=f2)
                         print("if [ $? -eq 0 ]; then", file=f2)
                         if mandc_report:
-                            print(
-                                "  add_rtp_process_event.py {} finished".format(
-                                    filename
-                                ),
-                                file=f2,
-                            )
+                            if len(obsid_list) > 1:
+                                print(
+                                    f"  add_rtp_process_event.py {filename} {action} "
+                                    f"finished --file_list {obsid_list_str}",
```

Comment:
I suppose this `obsid_list_str` will exist because of python magic right? It gets defined in line 1112. I think I would just feel better if it got re-computed here so we know explicitly it is a thing. or maybe have it outside of both if statements? Just seems like something someone will come back to in the future and go "where is this variable from?"

Reply:
Good point, I can change that